### ISSUE TITLE
[3pt] Block validation error

### ISSUE THREAD

**Issue by Tuxman** [Topl/Bifrost#362]
The query nodes on the Valhalla testnet are receiving blocks and failing to append blocks due to a difficulty score error:

Part of Topl/Bifrost#358

```[hDaBqlh0OIy1ToDL-akka.actor.default-dispatcher-639] bifrost.history.BifrostHistory - History: block 5NxwFNCW8Kt78Ea9KZjVEMZW2QZZsfBtJYbB6C1eepQ1 appended to chain with score Some(50534753510993000). Best score is 50534753510993000. Pair: 5NxwFNCW8Kt78Ea9KZjVEMZW2QZZsfBtJYbB6C1eepQ1
14:19:24.947| WARN  [hDaBqlh0OIy1ToDL-akka.actor.default-dispatcher-639] bifrost.history.BifrostHistory - Block validation failed
java.lang.IllegalArgumentException: requirement failed: 43697233562301855 < 171149456 failed, 26711846800,
        at scala.Predef$.require(Predef.scala:281)
        at bifrost.validation.DifficultyBlockValidator.$anonfun$checkConsensusRules$1(DifficultyBlockValidator.scala:22)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
        at scala.util.Try$.apply(Try.scala:213)
        at bifrost.validation.DifficultyBlockValidator.checkConsensusRules(DifficultyBlockValidator.scala:16)
        at bifrost.validation.DifficultyBlockValidator.validate(DifficultyBlockValidator.scala:12)
        at bifrost.validation.DifficultyBlockValidator.validate(DifficultyBlockValidator.scala:10)
        at bifrost.history.BifrostHistory.$anonfun$append$2(BifrostHistory.scala:79)
        at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:273)
        at scala.collection.immutable.List.foreach(List.scala:392)
        at scala.collection.TraversableLike.map(TraversableLike.scala:273)
        at scala.collection.TraversableLike.map$(TraversableLike.scala:266)
        at scala.collection.immutable.List.map(List.scala:298)
        at bifrost.history.BifrostHistory.$anonfun$append$1(BifrostHistory.scala:79)
        at scala.util.Try$.apply(Try.scala:213)
        at bifrost.history.BifrostHistory.append(BifrostHistory.scala:76)
        at bifrost.history.BifrostHistory.append(BifrostHistory.scala:32)
        at bifrost.scorexMod.GenericNodeViewHolder.bifrost$scorexMod$GenericNodeViewHolder$$pmodModify(GenericNodeViewHolder.scala:94)
        at bifrost.scorexMod.GenericNodeViewHolder$$anonfun$processRemoteModifiers$1.$anonfun$applyOrElse$14(GenericNodeViewHolder.scala:210)
        at bifrost.scorexMod.GenericNodeViewHolder$$anonfun$processRemoteModifiers$1.$anonfun$applyOrElse$14$adapted(GenericNodeViewHolder.scala:210)
        at scala.Option.foreach(Option.scala:407)
        at bifrost.scorexMod.GenericNodeViewHolder$$anonfun$processRemoteModifiers$1.$anonfun$applyOrElse$8(GenericNodeViewHolder.scala:210)
        at bifrost.scorexMod.GenericNodeViewHolder$$anonfun$processRemoteModifiers$1.$anonfun$applyOrElse$8$adapted(GenericNodeViewHolder.scala:188)
        at scala.Option.foreach(Option.scala:407)
        at bifrost.scorexMod.GenericNodeViewHolder$$anonfun$processRemoteModifiers$1.applyOrElse(GenericNodeViewHolder.scala:188)
        at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:175)
        at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:176)
        at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:176)
        at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:176)
        at akka.actor.Actor.aroundReceive(Actor.scala:539)
        at akka.actor.Actor.aroundReceive$(Actor.scala:537)
        at bifrost.BifrostNodeViewHolder.aroundReceive(BifrostNodeViewHolder.scala:22)
        at akka.actor.ActorCell.receiveMessage(ActorCell.scala:612)
        at akka.actor.ActorCell.invoke(ActorCell.scala:581)
        at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:268)
        at akka.dispatch.Mailbox.run(Mailbox.scala:229)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:241)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)```

**Comment by Tuxman** [comment]
The query nodes were trying to forge, and getting bad hit values because they do not have complete wallet data to forge properly.

**Comment by scasplte2** [comment]
How was this resolved @Tuxman ?

**Comment by Tuxman** [comment]
It was fixed in commit cc15dfd, I referenced #368 but did not reference this one also in the commit.
